src/Kirel.Identity.DTOs/KirelUserAuthenticationDto.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Kirel.Identity.DTOs;
+
+/// <summary>
+/// Dto for user authentication
+/// </summary>
+public class KirelUserAuthenticationDto
+{
+    /// <summary>
+    /// Authentication type. Must be one of the following: Username, Phone, Email
+    /// </summary>
+    public string Type { get; set; } = "";
+    /// <summary>
+    /// User unique name
+    /// </summary>
+    public string Username { get; set; } = "";
```

Comment:
Can you simply use a single field named "Login" and with a case switch for the type (for example), define the login type? This will help you maintain some backward compatibility